test/test_config.py

```diff
@@ -238,6 +240,111 @@ def check_test_value(cr, value):
         with GitConfigParser(fpa, read_only=True) as cr:
             check_test_value(cr, tv)
 
+    @with_rw_directory
+    def test_conditional_includes_from_git_dir(self, rw_dir):
+        # Initiate repository path
+        git_dir = osp.join(rw_dir, "target1", "repo1")
+        os.makedirs(git_dir)
+
+        # Initiate mocked repository
+        repo = mock.Mock(git_dir=git_dir)
```

Comment:
So nice to see a mock for once. It's my first time in Python.

Reply:
I still mostly code in a Python 2.7 environment so I didn't know that mock was added to the unittest module, very nice addition :)
https://docs.python.org/3/library/unittest.mock-examples.html